includes/admin/reporting/reports.php

```diff
@@ -219,10 +219,18 @@ function give_reports_forms_table() {
 	}
 
 	include( dirname( __FILE__ ) . '/class-form-reports-table.php' );
-
-	$give_table = new Give_Form_Reports_Table();
-	$give_table->prepare_items();
-	$give_table->display();
+	?>
+	<form class="give-forms-reports-filter" method="get">
```

Comment:
@mehul0810 Instead of directly putting `form` tag around the table, We can  achieve this by setting api because these pages are generated by setting api

For ref:
https://github.com/WordImpress/Give/blob/master/includes/admin/reporting/class-settings-forms.php#L53
https://github.com/WordImpress/Give/blob/master/includes/admin/views/html-admin-settings.php#L21